microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -9,9 +9,9 @@ ms.prod: microsoft-edge
 ms.technology: webview
 keywords: WebView2, webview2, WebView, webview, winui apps, winui, edge, CoreWebView2, browser control, edge html, get started, Get Started, .NET
 ---
-# Get started with WebView2 in WinUI 2 apps (public preview)
+# Get started with WebView2 in WinUI 2 (UWP) apps (public preview)
```

Comment:
Also change the "title:" field in the YAML block at top of file to match.

Reply:
done